workflow-testing/src/main/java/com/squareup/workflow1/testing/RealRenderTester.kt

```diff
@@ -101,25 +92,44 @@ internal class RealRenderTester<PropsT, StateT, OutputT, RenderingT>(
     ) : Expectation<Nothing>() {
       override fun describe(): String = description
     }
+
+    data class ExpectedRemember(
+      val matcher: (RememberInvocation) -> Boolean,
+      val exactMatch: Boolean,
+      val description: String,
+    ) : Expectation<Any?>() {
```

Comment:
Why is this `Expectation<Any?>`? Remembered expectations are closer to side effects so I'd expect `Expectation<Nothing>`. This matches the workflow expectation format which is maybe ok too? I don't really know the distinction here

Reply:
Oh right. That was right around the time I realized I couldn't provide a mock value. Thanks, good catch.